components/kyma-environment-broker/internal/process/upgrade_kyma/initialisation.go

```diff
@@ -64,6 +68,14 @@ func (s *InitialisationStep) Name() string {
 func (s *InitialisationStep) Run(operation internal.UpgradeKymaOperation, log logrus.FieldLogger) (internal.UpgradeKymaOperation, time.Duration, error) {
 	if operation.State == orchestrationExt.Canceled {
 		log.Infof("Skipping processing because orchestration %s was canceled", operation.OrchestrationID)
+
+		// revert to original evaluation status
+		operation, _, err := s.internalEvalUpdater.RestoreStatusToEval(operation, log)
```

Comment:
state can only be canceled in rare condition situation when the operation was pending (not in progress) and the operation started to be executed concurrently with the cancellation. So this code is not needed